homeassistant/components/melcloud/climate.py

```diff
@@ -181,6 +230,22 @@ def fan_modes(self) -> Optional[List[str]]:
         """Return the list of available fan modes."""
         return self._device.fan_speeds
 
+    async def async_set_vane_horizontal(self, position: str) -> None:
+        """Set horizontal vane position."""
+        if position not in self._device.vane_horizontal_positions:
+            raise ValueError(
+                f"Unsupported horizontal vane position {position}. Check the available modes from state attributes."
```

Comment:
You could log the available positions here as well so it is easier for the user to correct their mistake.